posthog/batch_exports/http.py

```diff
@@ -356,6 +357,7 @@ class Meta:
         ]
         read_only_fields = ["id", "team_id", "created_at", "last_updated_at", "latest_runs", "schema"]
 
+    # TODO: could this be moved inside BatchExportDestinationSerializer::validate?
```

Comment:
There is some instance stuff that makes this not straight-forward, but maybe we can access the same objects via some context or something. I would be okay with moving the method or part of the method over to `BatchExportDestinationSerializer::validate`. 

Regardless, this is not important, just a thought.

Reply:
yeah, not a biggie, but it's a bit confusing having 2 methods for validating the destination. Maybe one to refactor later